ngraph/core/src/op/and.cpp

```diff
@@ -7,6 +7,8 @@
 #include "ngraph/runtime/host_tensor.hpp"
 #include "ngraph/runtime/reference/and.hpp"
 
+#include "ngraph/validation_util.hpp"
```

Comment:
As I see here we have evaluate method for different types which are not used if we can create operation only with boolean input/output. I propose to remove redundant types

Reply:
Of course. Removed